fine_tune.py

```diff
@@ -0,0 +1,226 @@
+"""
+This file contains additional code related to fine tuning, here you will find functions that load annotation data and starts running fine-tuning on the selected model.
+"""
+
+import pandas as pd
+import numpy as np
+import torch
+import torch.utils
+import torch.utils.data
+import json
+import os
+
+import datasets
+import losses
+import setup
+import models
+import train
+
+def load_annotation_data(annotation_file, taxa_of_interest):
+    """
+    load_inat_data copy for loading annotation data extracted from the database.
+
+    Parameters:
+        - annotation_file: the filename for the csv that has the annotations wanted to fine tune
+        - taxa_of_interest: list taxa that you want to filter annotations by
+    
+    Returns:
+        - locs: location tuples of longitude and latitude
+        - taxa: list of taxa values in data
+        - hex_types: denotes absence or presence in a corresponding location
+    """
+
+    print('\nLoading ' + annotation_file)
+    data = pd.read_csv(annotation_file, index_col=0)
+
+    # remove outliers
+    num_obs = data.shape[0]
+    data = data[((data['latitude'] <= 90) & (data['latitude'] >= -90) & (data['longitude'] <= 180) & (data['longitude'] >= -180) )]
```

Comment:
Whoa! It's so funny that this has to be done as a data cleaning step. I wonder what's going on with the input data that this is necessary. (No changes needed, just expressing my shock and horror!)